### PR TITLE
clean_aws_resources: log exception from wrapped method

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -123,6 +123,7 @@ def wrapper(*args, **kwargs):
         try:
             return method(*args, **kwargs)
         except Exception:
+            TEST_LOG.exception('Exception in wrapped method %s', method.__name__)
```
Comment (lmr) [inline sdcm/tester.py]: If you are logging that an exception happened, you might as well log the exception type.

Comment (larisau) [inline sdcm/tester.py]: logging exception method always prints Exception name + traceback

Comment (lmr) [inline sdcm/tester.py]: Yep, my mistake, my brain read `error` instead of `exception`. Disregard the original comment.

Comment (larisau) [inline sdcm/tester.py]: I got it, it's what usually happens to all :)

**Pull request merged**